tools/windows/install.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/env bash
+
+# install msys2 to use this @ http://www.msys2.org
+
+CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TOP=$(readlink -f $CWD/../..)
+DWNLDD=$TOP/downloaded
+
+P_MAJ="3"
+P_MIN="6"
+P_VER="$P_MAJ.$P_MIN.2"
+NUMKL_VER="1.13.1"
+NUMDC_VER="0.7.0"
+SCIPY_VER="0.19.1"
+
+PPATH=0
+SPATH=0
+
+CURL=0
+which curl.exe &> /dev/null && CURL=$(which curl.exe)
+[ $CURL == 0 ] && echo "curl not found, cannot download installers"
+
+source $CWD/check_python.sh
+source $CWD/check_swig.sh
+
+PIP=$(dirname $PPATH)/Scripts/pip.exe
+[ ! -f $PIP ] && echo "ERROR: PIP not found @ $PIP" && exit 1
+
+
+SO=$($PPATH -c "import distutils; from distutils import sysconfig; print(sysconfig.get_config_var('SO'))")
+WHEEL="${SO:6:-4}"
+
+NUMLK="numpy-$NUMKL_VER+mkl-cp${P_MAJ}${P_MIN}-cp${P_MAJ}${P_MIN}m-${WHEEL}.whl"
+NUMDC="numpydoc-$NUMDC_VER-py2.py3-none-any.whl"
+SCIPY="scipy-$SCIPY_VER-cp${P_MAJ}${P_MIN}-cp${P_MAJ}${P_MIN}m-${WHEEL}.whl"
+
+if [ ! -f "$DWNLDD/$NUMLK" ] || [ ! -f "$DWNLDD/$NUMDC" ] || [ ! -f "$DWNLDD/$SCIPY" ]; then
+    echo "please download the following files to $DWNLDD and rerun this script"
+    echo $NUMLK
+    echo $NUMDC
+    echo $SCIPY    
+    echo "From: http://www.lfd.uci.edu/~gohlke/pythonlibs"
+    exit 1
+fi
+
+$PIP install wheel
+$PIP install $DWNLDD/$NUMLK
+$PIP install $DWNLDD/$NUMDC
+$PIP install $DWNLDD/$SCIPY
+
+$PIP install -r $TOP/requirements.txt
+$PPATH -m pip install --upgrade setuptools
+
+echo ""
+echo "Attempting to compile: vcvars.bat must be called manually prior to this"
```

Comment:
Installing Visual Studio doesn't always configure the paths correctly, so it's a good idea to call the "vcvars.bat" file which sets up the environment for compiling.